app/views/components/linkButton.scala.html

```diff
@@ -0,0 +1,23 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(url: String, value: String, classes: String = "govuk-button", id: Option[String] = None)(implicit messages: Messages)
+
+   <a href=@url  role="button" draggable="false" class=@classes data-module="govuk-button">
+    @messages(value)
```

Comment:
I think I would pass in the value I want to display rather than forcing this to be a message file key. The implicit messages wouldn't be needed then.